packages/expo/package.json

```diff
@@ -10,7 +10,11 @@
     "*.fx.web.tsx",
     "./src/winter/*.ts"
   ],
-  "bin": "bin/cli",
+  "bin": {
+    "expo": "bin/cli",
+    "expo-modules-autolinking": "bin/autolinking",
```

Comment:
This may actually collide with https://github.com/expo/expo/blob/411cd58e38fad5a482fe61ddf16f631e2df84137/packages/expo-modules-autolinking/package.json#L17C6-L17C30

Not sure if that happens within the `node_modules/expo` folder (it shouldn't).

Reply:
yes, also fingerprint will have collision. it won't hurt to use whatever bins, both bins should work.